README.md

```diff
@@ -170,6 +170,7 @@ Hurray! You successfully made a contribution! 🎉
   - [Aniket Singh](https://github.com/aniketsingh98571)
   - [Animesh Pathak](https://github.com/sonichigo)
   - [Anirudh Eyyani](https://github.com/AnirudhEyyani)
+  - [Aniruddha Inge](https://github.com/Aniruddha-Inge)
```

Comment:
Please add your name in line 172.

Reply:
ok I will change in soon. Sorry for inconvenience.

Reply:
![error](https://user-images.githubusercontent.com/91176720/134965214-94291b0a-8a8a-4b7e-9429-fbd3b0b2164a.jpg)

I am facing an error saying denied permission after using "git push origin aniruddha1" command. I tried many times but every single time this same error popped up. Could you please tell me the solution of it?

Reply:
You must push your changes to your fork, not to the original repository.
To do this quicker (instead of going to your fork and update), you can run

```bash
git push --repo https://github.com/Aniruddha-Inge/hacktoberfest-practice/
```

Let me know if that works. Thank you. 🙂

Reply:
![image](https://user-images.githubusercontent.com/91176720/134967614-3295018c-05f3-492f-ab77-90b0fe75c34a.png)

showing "everything up-to-date " :(   Should I start the same process from the beginning and create a new pull request?

Reply:
> ![image](https://user-images.githubusercontent.com/91176720/134967614-3295018c-05f3-492f-ab77-90b0fe75c34a.png)
> 
> showing "everything up-to-date " :( Should I start the same process from the beginning and create a new pull request?

Try to clone your fork (not the main repository) implement your changes, then push your changgs. 🙂

Reply:
![image](https://user-images.githubusercontent.com/91176720/134970972-d82708ff-213e-426d-857e-03c5baf8b557.png)

I'm sorry to disturb you again and again but still it is not getting pushed :(

Reply:
> ![image](https://user-images.githubusercontent.com/91176720/134970972-d82708ff-213e-426d-857e-03c5baf8b557.png)
> 
> I'm sorry to disturb you again and again but still it is not getting pushed :(

Looks like your branch is outdated. Try to make the changes directly in GitHub, then.

Reply:
@Aniruddha-Inge now you have to do `git pull origin main` as your clone is behind. Any more help, feel free to ask.

This happens as your branch copy is behind in others commits, which were merged. So if you put the following command in, it should work, and get all the other commits, and then try commiting again. Thanks.

Reply:
> 
> 
> > ![image](https://user-images.githubusercontent.com/91176720/134970972-d82708ff-213e-426d-857e-03c5baf8b557.png)
> > I'm sorry to disturb you again and again but still it is not getting pushed :(
> 
> Looks like your branch is outdated. Try to make the changes directly in GitHub, then.

When I get this, I just git pull, and then commit, and it works, unless this is something else.

Reply:
> > > ![image](https://user-images.githubusercontent.com/91176720/134970972-d82708ff-213e-426d-857e-03c5baf8b557.png)
> > > I'm sorry to disturb you again and again but still it is not getting pushed :(
> > 
> > 
> > Looks like your branch is outdated. Try to make the changes directly in GitHub, then.
> 
> When I get this, I just git pull, and then commit, and it works, unless this is something else.

That could mess up their branch as they already have committed changes, I guess.

Reply:
I just get this: 
![image](https://user-images.githubusercontent.com/70807500/134972858-79c9a1a9-9499-4780-bf7b-3fbc09e26f55.png)
but if it doesnt work, then like @Panquesito7 said, just do it directly to GitHub